AnkiHubSoftware/ankihub_addon
# [BUILD-372] Improve error message when users try to suggest changes to protected fields

## Related issues

https://ankihub.atlassian.net/browse/BUILD-372


## Proposed changes

This PR tries to improve the error message shown to users when they submit suggestions with no changes or only changes to protected fields/tags.


## How to reproduce

- Protect some field from the website.
- Change something in the protected field in some note in Anki and submit a suggestion
- Notice the error message.

## Screenshots and videos

![image](https://github.com/ankipalace/ankihub_addon/assets/41397710/88e2e199-1466-45ce-8b73-5231258f3d0e)
